examples/source-to-image/README.md

```diff
@@ -0,0 +1,159 @@
+This example provides scripts for building custom Jupyter Notebook images containing notebooks, data files, and with Python packages required by the notebooks already installed. The scripts provided work with the Source-to-Image tool and you can create the images from the command line on your own computer. It provides similar capabilities to ``mybinder.org`` but without the complexity of having to setup any special infrastructure. Templates are also provided to enable running builds in OpenShift, as well as deploying the resulting image to make it available.
```

Comment:
I don't think this jab at mybinder.org is useful or accurate. What it's more comparable to is one *piece* of Binder, repo2docker, which also requires nothing but python and docker.